lib/catalog/survey_compare.rb

```diff
@@ -0,0 +1,25 @@
+module Catalog
+  class SurveyCompare
+    attr_reader :base
+    class << self
+      def changed?(plan)
+        potential = new(plan)
+        encoded_base = Base64.strict_encode64(potential.base['create_json_schema'].to_json)
```

Comment:
So currently the base is actually what `create_json_schema` contains (same for modified). Easiest way would probably be just to wait until #452 gets merged then re-run your specs to see where we didn't line up. Unless I'm looking at it wrong there where base/modified should just hold the entire rendered page rather than just `create_json_schema`